HabitLogger.JaegerByte/Program.cs

```diff
@@ -0,0 +1,222 @@
+        }
+
+        static void GetUserInput()
+        {
+            ConsoleKey userInput;
+            while (true)
+            {
+                userInput = Console.ReadKey(true).Key;
+                switch (userInput)
+                {
+                    case ConsoleKey.D1:
+                        Console.Clear();
+                        InsertLog();
+                        break;
+                    case ConsoleKey.D2:
+                        Console.Clear();
+                        DeleteLog();
+                        break;
+                    case ConsoleKey.D3:
+                        Console.Clear();
+                        UpdateLog();
+                        break;
+                    case ConsoleKey.D4:
+                        Console.Clear();
+                        GetLog();
+                        break;
+                    case ConsoleKey.D0:
+                        System.Environment.Exit(0);
+                        break;
+                }
+                break;
+            }
+        }
+
+        static void InsertLog()
+        {
+            Console.WriteLine("New log:");
+            Console.WriteLine("Please insert the date (dd-mm-yyyy) and confirm with ENTER");
+            string inputDate = Console.ReadLine();
+            Console.WriteLine("Please insert the quantity als whole number and confirm with ENTER");
+            int inputQuantity;
+            Int32.TryParse(Console.ReadLine(), out inputQuantity);
+            DateTime result; // not used
+            if (DateTime.TryParseExact(inputDate, "dd-MM-yyyy", new CultureInfo("de-DE"), DateTimeStyles.None, out result) && inputQuantity>0)
+            {
+                CommandInsertLog(inputDate, inputQuantity);
+                Console.WriteLine("log inserted successfully!");
+                Console.WriteLine("press ANY key to get back to the menu");
+                Console.ReadKey(true);
+            }
+            else
+            {
+                Console.WriteLine("invalid input!");
+                Console.WriteLine("press ANY key to get back to the menu");
+                Console.ReadKey(true);
+            }
+        }
+
+        static void CommandInsertLog(string date, int quantity)
+        {
+            SqliteCommand insertCommand = connection.CreateCommand();
+            insertCommand.CommandText = $"INSERT INTO pushups(Date, Quantity) VALUES('{date}', {quantity})";
+            connection.Open();
+            insertCommand.ExecuteNonQuery();
+            connection.Close();
+        }
+
+        static void DeleteLog()
+        {
+            Console.WriteLine("Delete log:");
+            Console.WriteLine("Please insert Id and confirm with ENTER");
+            int inputIndex;
+            if (Int32.TryParse(Console.ReadLine(), out inputIndex))
+            {
+                CommandDeleteLog(inputIndex);
+                Console.WriteLine("log deleted successfully!");
```

Comment:
You might want to make it such that this message is only printed when a log is actually deleted. You can do this by returning something from your CommandDeleteLog method or by checking that the ID exists and then printing the message. This isn't strictly necessary for the requirements, but I think it would be a significant improvement.

Reply:
Thank you for your input. I will fix this